pages/_document.tsx

```diff
@@ -0,0 +1,32 @@
+import Document, { Head, Html, Main, NextScript } from 'next/document'
+import CookieBanner from '@components/CookieBanner'
+import Footer from '@components/Footer'
+import Header from '@components/Header'
+import javascriptHash from '../public/assets/javascript/manifest.json'
+
+class GovukTemplate extends Document {
+  static async getInitialProps (ctx) {
+    const initialProps = await Document.getInitialProps(ctx)
```

Comment:
Couldn't this be called `context`?
I'm sure this is probably some standard I'm not aware of yet, so please ignore me!

Reply:
going by their Custom Documents docs https://github.com/vercel/next.js/blob/canary/docs/advanced-features/custom-document.md#custom-document

Reply:
got it, many thanks!